hyperspy/_signals/eds_tem.py

```diff
@@ -242,3 +245,171 @@ def get_calibration_from(self, ref, nb_pix=1):
         if mp_ref.has_item("Detector.EDS.live_time"):
             mp.Acquisition_instrument.TEM.Detector.EDS.live_time = \
                 mp_ref.Detector.EDS.live_time / nb_pix
+
+    def quantification_cliff_lorimer(self,
+                                     intensities,
+                                     kfactors,
+                                     composition_units='weight',
+                                     navigation_mask=1.0,
+                                     closing=True,
+                                     plot_result=False,
+                                     **kwargs):
+        """
+        Quantification using Cliff-Lorimer
+
+        Parameters
+        ----------
+        intensities: list of signal
+            the intensitiy for each X-ray lines.
+        kfactors: list of float
+            The list of kfactor in same order as intensities. Note that
+            intensities provided by hyperspy are sorted by the aplhabetical
+            order of the X-ray lines. eg. kfactors =[0.982, 1.32, 1.60] for
+            ['Al_Ka','Cr_Ka', 'Ni_Ka'].
+        composition_units: 'weight' or 'atomic'
+            Cliff-Lorimer return weight percent. By choosing 'atomic', the
+            return composition is in atomic percent.
+        navigation_mask : None or float or boolean numpy.array
+            The navigation locations marked as True are not used in the
+            quantification. If int is given the vacuum_mask method is used to
+            generate a mask with the int value as threhsold.
+        closing: bool
+            If true, applied a morphologic closing to the mask obtained by
+            vacuum_mask.
+        plot_result : bool
+            If True, plot the calculated composition. If the current
+            object is a single spectrum it prints the result instead.
+        kwargs
+            The extra keyword arguments for plotting. See
+            `utils.plot.plot_signals`
+
+        Return
+        ------
+        A list of quantified elemental maps giving the composition of the
+        sample in weight or atomic percent.
+
+        Examples
+        ---------
+        >>> #s is a signals.EDSTEMSpectrum
+        >>> s.set_elements(["Al", "Cr", "Ni"])
+        >>> s.add_lines()
+        >>> kfactors = [0.982, 1.32, 1.60]
+        >>> intensities = s.get_lines_intensity()
+        >>> res = s.quantification_cliff_lorimer(intensities,kfactors)
+
+        See also
+        --------
+        vacuum_mask
+        """
+        if isinstance(navigation_mask, float):
+            navigation_mask = self.vacuum_mask(navigation_mask, closing).data
+        elif navigation_mask is not None:
+            navigation_mask = navigation_mask.data
```

Comment:
That means the `navigation_mask` is a signal? In the docs it says it's a np.array....